modules/mod_acl_user_groups/tests/mod_acl_user_groups_tests.erl

```diff
@@ -0,0 +1,30 @@
+%% @doc Tests for mod_acl_user_groups
+-module(mod_acl_user_groups_tests).
+
+-include_lib("eunit/include/eunit.hrl").
+-include("zotonic.hrl").
+
+person_can_edit_own_resource_test() ->
+    Context = z_context:new(testsandboxdb),
+    z_module_manager:activate(mod_content_groups, Context),
+    z_module_manager:activate(mod_acl_user_groups, Context),
+
+    %% Person must be able to edit person category
+    m_acl_rule:replace_managed(
+        [
+            {rsc, [
+                {acl_user_group_id, acl_user_group_members},
+                {actions, [view, update]},
+                {is_owner, true},
+                {category_id, person}
+            ]}
+        ],
+        testsandboxdb,
+        z_acl:sudo(Context)
+    ),
```

Comment:
That is indeed a problem where the module is requested to start, but as it is starting async, it isn't finished starting when the test is running.

We might want to add something like `z_module_manager:await(ModuleName, Context)`.